src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -119,6 +119,41 @@ function _assignPanelId(element) {
   }
 }
 
+/**
+ * Check and warns if element has conflicting attributes.
+ * @param element Root element to check
+ * @param attribute An attribute that is conflicting with other attributes
+ * @param attrsConflictingWith The attributes conflicting with `attribute`
+ */
+function _warnConflictingAttributes(element, attribute, attrsConflictingWith) {
```

Comment:
Could we just take in a set of conflicting attributes and ensure that at most one attribute from the set is applied to the element? I'm thinking of the case where we might have more than two attributes that are conflicting (e.g. `light`, `seamless`, `dark`).

Reply:
@nbriannl thanks for pointing out that we have some attributes where conflicts are not transitive. Since we don't have a specific use case for dealing with a set of mutually conflicting attributes for now, I'm happy to stick with this approach for now.